modules/evm/src/lib.rs

```diff
@@ -313,6 +324,10 @@ decl_event! {
 		ContractDevelopmentDisabled(AccountId),
 		/// Deployed contract. \[contract\]
 		ContractDeployed(EvmAddress),
+		/// Seted contract code. \[contract\]
+		ContractSetedCode(EvmAddress),
```

Comment:
```suggestion
		ContractSetCode(EvmAddress),
```

There is no `Seted`, just `Set`